src/main.ts

```diff
@@ -45,6 +46,7 @@ import * as options from './options';
           const sourceMapOptions = {
             include: sourcemaps,
             projects: localProjects,
+            dist,
```

Comment:
this is where `INPUT_DIST` / `${ inputs.dist }` / `with: dist: 'foo-dist'`  is passed to the sentry sourcemap upload command.